Dockerfile.ubi

```diff
@@ -0,0 +1,77 @@
+FROM registry.redhat.io/ubi8/ubi
+
+EXPOSE 8080
+
+ENV PORT=8080 \
+    LOG_DIR=/opt/conjur-server/log \
+    TMP_DIR=/opt/conjur-server/tmp \
+    SSL_CERT_DIRECTORY=/opt/conjur/etc/ssl \
+    RAILS_ENV=production
+
+ENV RUBY_MAJOR_VERSION=2 \
+    RUBY_MINOR_VERSION=5
+
+ENV RUBY_VERSION="${RUBY_MAJOR_VERSION}.${RUBY_MINOR_VERSION}" \
+    RUBY_SCL_NAME_VERSION="${RUBY_MAJOR_VERSION}${RUBY_MINOR_VERSION}" \
+    RUBY_SCL="ruby-${RUBY_SCL_NAME_VERSION}"
+
+LABEL name="conjur-ubi" \
+      vendor="CyberArk" \
+      version="$VERSION" \
+      release="$VERSION" \
+      summary="Conjur UBI based container" \
+      description="Conjur UBI based container"
```

Comment:
```suggestion
      summary="Conjur UBI-based image" \
      description="Conjur provides secrets management and machine identity for modern infrastructure. "
```

Reply:
for the LICENSE part, I'm not sure the md format can work with the Redhat registry.
In our other Redhat repos we use a LICENSE which isn't with md extension and it looks a little bit different.
for example: https://github.com/cyberark/secretless-broker/blob/master/LICENSE

Reply:
other projects use Apache 2.0, but this project uses LGPL. I think we should move forward with the license as-is for now, and we can modify if needed when we start publishing to RH and find out we have issues. I don't see anything in their [docs](https://redhat-connect.gitbook.io/partner-guide-for-red-hat-openshift-and-container/certify-your-application/creating-a-container-application-project) that indicates that we should have an issue with a MD license file, fwiw.

Reply:
[this page](https://connect.redhat.com/zones/containers/container-certification-policy-guide) just says:
> How? Create a directory named /licenses and include all relevant licensing and/or terms and conditions as text file(s) in that directory

Reply:
not a ruby person so not sure this is the only change needed, but it solves the issue when building the image